packages/frontend/src/PortalMap/index.tsx

```diff
@@ -127,20 +134,21 @@ const PortalMap = () => {
                 'text-outline-width': 1,
                 'text-outline-opacity': 0.5,
                 'text-margin-y': -5,
-                shape:
-                  z.type.indexOf('TUNNEL_HIDEOUT') >= 0
-                    ? 'pentagon'
-                    : z.type.indexOf('TUNNEL_') >= 0
-                    ? 'cut-rectangle'
-                    : '',
+                shape: z.type.includes('TUNNEL_HIDEOUT')
```

Comment:
This should probably be extracted to a function

Reply:
fair